include/boost/geometry/algorithms/detail/overlay/check_enrich.hpp

```diff
@@ -26,6 +26,7 @@
 #include <boost/range/value_type.hpp>
 
 #include <boost/geometry/algorithms/detail/overlay/overlay_type.hpp>
+#include <boost/geometry/views/enumerate.hpp>
```

Comment:
this should be `#include <boost/geometry/views/enumerate_view.hpp>`

Reply:
Oh - that slipped through my cmake. Thanks!
I got mails about this too, so that works great.